OvmfPkg/Sec/SecMain.c

```diff
@@ -764,6 +764,13 @@ SecMtrrSetup (
     return;
   }
 
+ #if defined (TDX_GUEST_SUPPORTED)
+  if (CcProbe () == CcGuestTypeIntelTdx) {

```

Comment:
please add detailed comment on why it is skipped.

Reply:
Thanks, updated now